pkgs/by-name/ta/tailscale/package.nix

```
@@ -175,6 +175,46 @@ buildGoModule (finalAttrs: {
     in
     [ "-skip=^${builtins.concatStringsSep "$|^" skippedTests}$" ];
 
+  # copied from pkgs/build-support/go/module.nix, with modification
+  checkPhase = ''
```

```suggestion
  ${if stdenv.buildPlatform.isLinux then "checkPhase" else null} = ''
```

This makes the build on Darwin and cross work correctly, I think.